swaglyrics/cli.py

```diff
@@ -52,6 +56,12 @@ def stripper(song: str, artist: str) -> str:
 	url_data = re.sub(nlt, '', url_data)  # remove non-latin characters before unidecode
 	url_data = unidecode(url_data)  # convert accents and other diacritics
 	url_data = re.sub(aln, '', url_data)  # remove punctuation and other characters
+	if len(re.sub(spc, '', url_data.strip())) < 1:		 
+		return None
+	if url_data[0] == '-':
```

Comment:
This is what I was looking for, checking for initial and final `-` is correct. You could have used startswith and endswith